src/main/scala/com/github/unchama/seichiassist/listener/PlayerBlockBreakListener.scala

```diff
@@ -373,4 +421,22 @@ class PlayerBlockBreakListener(
     event.setCancelled(true)
     player.sendMessage(s"${RED}Y-59以下に敷かれたハーフブロックは破壊不可能です。")
   }
+
+  /**
+   * ブロック破壊時、「マナ切れブロック破壊停止設定」を取得する。
+   * マナ切れブロック破壊設定が `true` になっている場合、プレイヤーに破壊抑制メッセージを送信する。
+   * @param player マナ切れブロック破壊停止設定を取得するプレイヤー
+   */
+  def isBreakBlockManaFullyConsumed(player: Player): IO[Boolean] = {
```

Comment:
見た限り `PlayerBlockBreakListener` 内の定義でしか使われていないようなので `private` にして良いと思います！(私の見落としだったらそのままで大丈夫です！)

```suggestion
  private def isBreakBlockManaFullyConsumed(player: Player): IO[Boolean] = {
```

Reply:
見た限り`PlayerBlockBreakListener`でしか使ってないなかったのでprivateつけます